client/src/components/Calendar/TODO.txt

```diff
@@ -0,0 +1,6 @@
+EVERYTHING THAT STILL NEEDS TO BE DONE:
```

Comment:
Like I mentioned for the other thing, let's just file github issues for all these and delete the file.